diff --git a/chef/cookbooks/rabbitmq/recipes/rabbit.rb b/chef/cookbooks/rabbitmq/recipes/rabbit.rb
index be690cb52..9a5d9f4db 100644
--- a/chef/cookbooks/rabbitmq/recipes/rabbit.rb
+++ b/chef/cookbooks/rabbitmq/recipes/rabbit.rb
@@ -39,17 +39,6 @@ end
 
 include_recipe "rabbitmq::default"
 
-if ha_enabled
-  log "HA support for rabbitmq is enabled"
-  include_recipe "rabbitmq::ha"
-  # All the rabbitmqctl commands are local, and can only be run if rabbitmq is
-  # local
-  service_name = "rabbitmq"
-  only_if_command = "crm resource show #{service_name} | grep -q \" #{node.hostname} *$\""
-else
-  log "HA support for rabbitmq is disabled"
-end
-
 if node[:rabbitmq][:ssl][:enabled]
   ssl_setup "setting up ssl for rabbitmq" do
     generate_certs node[:rabbitmq][:ssl][:generate_certs]
